### PR TITLE
fix input parameter requirements

### PR DESCRIPTION
**Pull request by jandelgado**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,18 +47,16 @@ coverage:
...
     - name: Install Go
       if: success()
-      uses: actions/setup-go@v2-beta
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Calc coverage
       run: |
```
Comment (septs) [inline README.md]:
```suggestion
      run: go test -v -covermode=count -coverprofile=coverage.out
```

File: action.yml [hunk 2]
```diff
@@ -5,15 +5,15 @@ description: "convert golang coverage to lcov format"
 inputs:
   infile:
     description: "go coverage input file"
-    required: true
+    required: false
```
Comment (septs) [inline action.yml:8]: not suggested, it is need `required: true`

but because filled default value, for user is **optional**.

File: README.md [hunk 3]
```diff
@@ -47,18 +47,16 @@ coverage:
...
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Calc coverage
-      run: |
-        export PATH=$PATH:$(go env GOPATH)/bin
+      run: go test -v -covermode=count -coverprofile=coverage.out
         go test -v -covermode=count -coverprofile=coverage.out
```
Comment (septs) [inline README.md]: .... this line need remove ...

Comment (septs) [reply README.md]: image

github limit, i can't span

File: README.md [hunk 4]
```diff
@@ -47,18 +47,16 @@ coverage:
   steps:
     - name: Install Go
       if: success()
-      uses: actions/setup-go@v2-beta
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15.x
```
Comment (septs) [inline README.md:52]:
```suggestion
```
why not use setup-go automate version select?

File: README.md (hunk 3 above)
Comment (jandelgado) [reply README.md]: yes, done

File: README.md (hunk 4 above)
Comment (jandelgado) [reply README.md:52]: i prefer explicit versioning for reproducable builds

Comment (septs) [reply README.md:52]: in for user of copy code, not is good idea.

**Pull request merged**